Fixtures/iOS/Project.xcodeproj/xcshareddata/xcschemes/iOS.xcscheme

```diff
@@ -5,6 +5,24 @@
    <BuildAction
       parallelizeBuildables = "YES"
       buildImplicitDependencies = "YES">
+      <PreActions>
```

Comment:
Have you tested opening this scheme in Xcode and see if it changes the format? We want to keep the work achieved by https://github.com/xcodeswift/xcproj/pull/216. You may have to add something to the `attributesOrder` in  `AEXML+XcodeFormat.swift`

Reply:
These `.xcscheme` additions were generated by Xcode itself, not by hand. I added actions through the Xcode UI and these were the results. I'll take a look at #216, thanks.

Reply:
I made a small addition to `attributesOrder` in b8fdd76abfc33ec966792664cc683c7178bc5097. In each of the updated `xmlElement` implementations, the insertion of `PreActions` and `PostActions` child elements happens before other children are added, which matches Xcode's xml generation.